### PR TITLE
EffectType Enum

### PR DESCRIPTION
**Pull request by TheVoidNebula**
I think we could do with effects as an enum then just in the Player class. And also it would help newer developers to understand and find all different effect types instead of searching a long time for them (they are nowhere documented) ^^

### CODE REVIEW

**Comment by TheVoidNebula** [comment]
while being here, I also have another Idea. What about having data, global data. Like data you can set and get from things like doors and players. maybe you do not only want to get all players but also all doors? or like all ragdolls etc.

**Review by iRebbok — COMMENTED** [review]
I like this idea. But it requires correct documentation and API. So I don't see any API additions in this PR.

About global data: not understandable. If you are talking about something like a dictionary with keys for a player in their object - it won't be implemented. Use other ways.

**Comment by TheVoidNebula** [comment]
what exactly you mean by API additions? ^^

**Comment by iRebbok** [comment]
At least a small part of API that uses the enum.

**Comment by TheVoidNebula** [comment]
like this?

**Comment by iRebbok** [comment]
@Thundermaker300 do you have any docs suggestions/corrections? Review if you do 🙃

**Comment by TheVoidNebula** [comment]
thanks, iRebbok for fixing my messy code ^^

**Comment by TheVoidNebula** [comment]
I do not think he wants to add anything :) if so he would have already done it xD

**Comment by Thundermaker300** [comment]
Haven't really had the chance to look at it yet

File: Exiled.API/Enums/EffectType.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        /// </summary>
+        Burned,
+
+        /// <summary>
+        /// Blurs the player's screen when rotating.
+        /// </summary>
+        Concussed,
+
+        /// <summary>
+        /// Teleports thet player to the pocket dimension and drains health.
```
Comment (iRebbok) [inline Exiled.API/Enums/EffectType.cs:46]:
```suggestion
        /// Teleports the player to the pocket dimension and drains health.
```

**Pull request merged**